packages/core/src/utilities.ts

```diff
@@ -32,3 +32,25 @@ export function extractStyles(style: StyleProp<any>) {
 
   return { viewStyles, textStyles };
 }
+
+/**
+ * Merges a style object on top of another style object. In React Native,
+ * keys with undefined values in a style object will still override styles
+ * that appear earlier in a sequence. This avoids that problem.
+ */
+export function applyStyles(
```

Comment:
This lets us get rid of that `...(something ? { something } : {})` pattern.

As discussed in a previous PR, there doesn't seem to be a better way to do this. These all seem to _not_ work (i.e. they all result in `{ color: undefined }`

- `const mergedStyles = [{ color: "red" }, { color: undefined }]`
- `const mergedStyles = StyleSheet.compose({ color: "red" }, { color: undefined })`
- `const mergedStyles = StyleSheet.flatten([{ color: "red" }, { color: undefined }])`

Reply:
do you mind putting this as a comment in there as an example? i think it'll make more sense

Reply:
👍 sure thing